src/components/Extensibility/components-form/NumberRenderer.js

```diff
@@ -26,6 +27,9 @@ export function NumberRenderer({
     ['min', 'max'].map(prop => [prop, schema.get(prop)]),
   );
 
+  const disableOnEdit = schema.get('disableOnEdit');
+  console.log('NumbererRenderer editMode: ', editMode);
```

Comment:
```suggestion
```

Reply:
resolved in [48f0b71](https://github.com/kyma-project/busola/commit/48f0b71b11596637f2310e86fd4ef126052c4b8d)